lib/js_of_ocaml/dom_html.ml

```diff
@@ -557,6 +557,10 @@ and eventTarget = object ('self)
 
   method onpointerup : ('self t, pointerEvent t) event_listener writeonly_prop
 
+  method onbeforetoggle : ('self t, toggleEvent t) event_listener writeonly_prop
+
+  method ontoggle : ('self t, toggleEvent t) event_listener writeonly_prop
+
```

Comment:
There are similar methods related to event handlers in this file that have the `prop` type instead of `writeonly_prop`. In fact, this PR initially used the latter type. Is this new version correct?